cli.py

```diff
@@ -0,0 +1,227 @@
+    'Cree': 'cr',
+    'Welsh': 'cy',
+    'Danish': 'da',
+    'German': 'de',
+    'Divehi': 'dv',
+    'Dzongkha': 'dz',
+    'Greek, Modern (1453-)': 'el',
+    'English': 'en',
+    'Esperanto': 'eo',
+    'Estonian': 'et',
+    'Ewe': 'ee',
+    'Faroese': 'fo',
+    'Persian': 'fa',
+    'Fijian': 'fj',
+    'Finnish': 'fi',
+    'French': 'fr',
+    'Western Frisian': 'fy',
+    'Fulah': 'ff',
+    'Georgian': 'ka',
+    'Gaelic': 'gd',
+    'Irish': 'ga',
+    'Galician': 'gl',
+    'Manx': 'gv',
+    'Guarani': 'gn',
+    'Pushto': 'ps',
+    'Quechua': 'qu',
+    'Romansh': 'rm',
+    'Romanian': 'ro',
+    'Russian': 'ru',
+    'Sango': 'sg',
+    'Sanskrit': 'sa',
+    'Sinhala': 'si',
+    'Slovak': 'sk',
+    'Slovenian': 'sl',
+    'Northern Sami': 'se',
+    'Samoan': 'sm',
+    'Shona': 'sn',
+    'Sindhi': 'sd',
+    'Somali': 'so',
+    'Sotho, Southern': 'st',
+    'Spanish': 'es',
+    'Sardinian': 'sc',
+    'Serbian': 'sr',
+    'Swati': 'ss',
+    'Sundanese': 'su',
+    'Swahili': 'sw',
+    'Swedish': 'sv',
+    'Tahitian': 'ty',
+    'Tamil': 'ta',
+    'Tatar': 'tt',
+    'Telugu': 'te',
+    'Tajik': 'tg',
+    'Tagalog': 'tl',
+    'Thai': 'th',
+    'Tigrinya': 'ti',
+    'Tonga (Tonga Islands)': 'to',
+    'Tswana': 'tn',
+    'Tsonga': 'ts',
+    'Turkmen': 'tk',
+    'Turkish': 'tr',
+    'Twi': 'tw',
+    'Uighur': 'ug',
+    'Ukrainian': 'uk',
+    'Urdu': 'ur',
+    'Uzbek': 'uz',
+    'Venda': 've',
+    'Vietnamese': 'vi',
+    'Volapük': 'vo',
+    'Walloon': 'wa',
+    'Wolof': 'wo',
+    'Xhosa': 'xh',
+    'Yiddish': 'yi',
+    'Yoruba': 'yo',
+    'Zhuang': 'za',
+    'Zulu': 'zu',
+}
+
+@click.command()
+@click.option(
+    '-q',
+    '--query',
+    type=str,
+    required=True,
+    help='Search query string. Required'
+)
+
+@click.option(
+    '-s',
+    '--sort',
+    type=click.Choice(['top', 'latest'], case_sensitive=False),
+    required=False,
+    help='Rank results by "top" or "latest',
+)
+
+@click.option(
+    '--since',
+    type=str,
+    required=False,
+    help=(
+        'Filter results for posts after the specified datetime (inclusive). '
+        'Use ISO 8601 format: "YYYY-MM-DD" or full datetime. Uses "sortAt" timestamp.'
+    )
+)
+
+@click.option(
+    '--until',
+    type=str,
+    required=False,
+    help=(
+        'Filter results for posts before the specified datetime (not inclusive). '
+        'Use ISO 8601 format: "YYYY-MM-DD" or full datetime. Uses "sortAt" timestamp.'
+    )
+)
+
+@click.option(
+    '-m',
+    '--mentions',
+    type=str,
+    help='Filter posts mentioning the specified account (omit the @ symbol).'
+)
+
+@click.option(
+    '-a',
+    '--author',
+    type=str,
+    required=False,
+    help='Filter posts by the specified account (omit the @ symbol).'
+)
+
+@click.option(
+    '-l',
+    '--lang',
+    type=str,
+    required=False,
+    help=f'Filter posts by language.\n\nLanguage Options:\n\n {', '.join(list(lang_dict.keys()))}\n\n'
```

Comment:
For the format string to work you need to have the {} expression encapsulated within the quotation marks. 
For example:
```
name = "Caleb"
f"Hello my name is {name}"
```
This code is an example of using the format string correctly.

Your code is the equivalent of doing this:
```
f"Hello my name is {", "name}"
```
As your code stands now, it doesn't do that. 

For the join function call, which string do you want to connect the languages with? Once you figure out which string you want to use to join the list of languages, try replacing line 179 with this code block:
```
help=f'Filter posts by language.\n\nLanguage Options:\n\n {<insert_desired_joiner_string_here>.join(list(lang_dict.keys()))}\n\n'
```